resources/sphinx/sphinx/conf.py.in

```diff
@@ -0,0 +1,271 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath('.')) # in order to use the local modules
+sys.path.insert(0, os.path.abspath("@PROJECT_SOURCE_DIR@/python"))
+import textwrap
+
+# cmake parser custom module
+from docutils.parsers.rst import Directive
+from docutils import nodes
+from sphinx.util.nodes import set_source_info
+currentmode = "user"
+
+# AutoStructify for math in markdown
+import recommonmark 
+from recommonmark.transform import AutoStructify
+
+# on_rtd is whether we are on readthedocs.org, this line of code grabbed from docs.readthedocs.org
+on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'@PROJECT_NAME@'
+copyright = (u'Copyright (c) 2020, New York University and ' +
+             u'Max Planck Gesellschaft. License under the BSD 3-clause.')
+author = u'Maximilien Naveau'
+
+# The short X.Y version
+version = u'1.0'
+# The full version, including alpha/beta/rc tags
+release = u'1.0.0'
```

Comment:
Where is this information used?  If it is something that shows up in the documentation, it should not be hard-coded but rather depend on arguments or variables.